sdk/search/search-documents/src/serviceModels.ts

```diff
@@ -966,84 +894,79 @@ export interface SimpleField {
    * returned in a search result. You can disable this option if you don't plan to return the field
    * contents in a search response to save on storage overhead. This can only be set during index
    * creation and only for vector fields. This property cannot be changed for existing fields or set
-   * as false for new fields. If this property is set to `false`, the property `hidden` must be set to
-   * `true`. This property must be true or unset for key fields, for new fields, and for non-vector
-   * fields, and it must be null for complex fields. Disabling this property will reduce index
-   * storage requirements. The default is true for vector fields.
+   * as false for new fields. If this property is set as false, the property 'hidden' must be set to
+   * 'true'. This property must be false or unset for key fields, for new fields, and for non-vector
+   * fields. Disabling this property will reduce index storage requirements.
```

Comment:
```suggestion
   * 'true'. This property must be true or unset for key fields, for new fields, and for non-vector
   * fields. Disabling this property will reduce index storage requirements. The default is true for vector fields.
```

This is pulled from the spec, but the spec comment might be wrong. Some properties have had their default flipped without being renamed. 

`hidden` is the negation of the spec's `retrievable`, so that bit is correct.